docs/en/observability/synthetics-private-location.asciidoc

```diff
@@ -125,4 +126,23 @@ When the {agent} is running you can add a new private location in {kib}:
 
 Now you can add monitors to your private location in <<synthetics-get-started-ui,the {uptime-app}>> or using the <<synthetics-get-started-project,Elastic Synthetics library's `push` method>>.
 
-TIP: You can run the same monitor both from Elastic's global managed infrastructure _and_ your private locations to compare results.
\ No newline at end of file
+TIP: You can run the same monitor both from Elastic's global managed infrastructure _and_ your private locations to compare results.
+
+[discrete]
+[[synthetics-private-location-scaling]]
+= Scaling private locations
+
+By default private locations are configured to allow two simultaneous browser tests, and an unlimited number of lightweight checks. 
+These limits can be set via the environment variables `SYNTHETICS_LIMIT_{TYPE}`, where `{TYPE}` is one of `BROWSER`, `HTTP`, `TCP`, and `ICMP` 
+for the container running the {agent} docker image. 
+
+It is critical to allocate enough memory and CPU capacity to handle configured limits. 
+Start by allocating at 2 GiB of memory and two cores per browser instance to ensure consistent
+performance and avoid out-of-memory errors. Then adjust as needed. Resource requirements will vary depending on workload. 
+Much less memory is needed for lightweight monitors. Start by allocating at 512MiB of memory and two cores for 
```

Comment:
Same? (at _least_...)

Reply:
```suggestion
Much less memory is needed for lightweight monitors. Start by allocating at least 512MiB of memory and two cores for 
```

Reply:
I really wonder how I made this same typo twice 🤔